src/bin/casr-cluster.rs

```diff
@@ -442,7 +514,24 @@ fn main() -> Result<()> {
                 .min_values(1)
                 .max_values(2)
                 .value_name("INPUT_DIR> <OUTPUT_DIR")
-                .help("Deduplicate CASR reports. If two directories are set, deduplicated reports are copied to the second directory. If one directory is provided, duplicated reports are deleted."),
+                .help(
+                    "Deduplicate CASR reports. If two directories are set, \
+                    deduplicated reports are copied to the second directory. \
+                    If one directory is provided, duplicated reports are deleted.",
+                ),
+        )
+        .arg(
+            Arg::new("merge")
+                .short('m')
+                .long("merge")
+                .takes_value(true)
+                .min_values(2)
+                .max_values(2)
+                .value_names(&["NEW_DIR", "STORAGE_DIR"])
+                .help(
+                    "Merge NEW_DIR into STORAGE_DIR. Only new CASR reports from \
```

Comment:
input/output gives a direct understanding of file copy source and destination

Reply:
done